src/app/examples/form/components/form.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <h1 class="main-heading">{{ 'anms.examples.form.title' | translate }}</h1>
+    </div>
+  </div>
+  <form [formGroup]="form" (ngSubmit)="onSubmit()">
+    <div class="row justify-content-center">
+      <div class="col-md-6" [ngClass]="routeAnimationsElements">
+        <mat-slide-toggle formControlName="autosave">{{ 'anms.examples.form.autosave' | translate }}</mat-slide-toggle>
+        <h2>{{ 'anms.examples.form.subtitle1' | translate }}</h2>
+        <div class="row">
+          <mat-form-field class="col" [ngClass]="routeAnimationsElements">
+            <input matInput placeholder="{{ 'anms.examples.form.placeholder1' | translate }}" formControlName="username" required>
```

Comment:
Is there a reason to put `required` in the inputs?
Because we're using reactive forms, we don't need to I think but I might be missing something